test/registry/protocol_v2.py

```diff
@@ -359,7 +359,7 @@ def history_for_image(image):
             "history": [history_for_image(image) for image in images],
         }
 
-        if images[-1].config:
+        if images and images[-1].config:
```

Comment:
Can you explain what this is doing?

Reply:
Sure. It is grabbing the config of the top layer in the history for use as the overall config of the manifest. V1 had config for each image (since each layer is its own image), while V2 only has config at the most recent layer.

Reply:
So there is no config for empty manifests?

Reply:
There is, this is just part of the test and is necessary for us to override the default empty one